packaging/systemd/setup_fog.sh

```diff
@@ -6,5 +6,18 @@
 # Export global environment variables
 export DRONE_DEVICE_ID
 
+# Enable SROS
+DIR="/enclave/sros"
+if [ -d "$DIR" ]; then
+  echo "Folder ${DIR} exists. Enabling SROS2."
+  export ROS_SECURITY_ENABLE=true
+  export ROS_SECURITY_STRATEGY=Enforce
+  export ROS_SECURITY_KEYSTORE="${DIR}"
+  export ROS_SECURITY_LOG_VERBOSITY=WARN
+  export ROS_SECURITY_LOG_FILE="${DIR}/log/ros-security.log"
```

Comment:
who writes the security log? if it is the ros nodes, they don't have rights to write into /enclave/ , as they are being executed as user "sad"